pkgs/development/python-modules/clifford/default.nix

```diff
@@ -22,12 +22,15 @@ buildPythonPackage rec {
     inherit pname version;
     sha256 = "ade11b20d0631dfc9c2f18ce0149f1e61e4baf114108b27cfd68e5c1619ecc0c";
   };
-  patches = [ (fetchpatch {
-    # Compatibility with h5py 3.
-    # Will be included in the next releasse after 1.3.1
-    url = "https://github.com/pygae/clifford/pull/388/commits/955d141662c68d3d61aa50a162b39e656684c208.patch";
-    sha256 = "00m8ias58xycn5n78sy9wywf4wck1v0gb8gzmg40inzdiha93jyz";
-  }) ];
+
+  patches = [
+    (fetchpatch {
+      # Compatibility with h5py 3.
+      # Will be included in the next releasse after 1.3.1
+      url = "https://github.com/pygae/clifford/pull/388/commits/955d141662c68d3d61aa50a162b39e656684c208.patch";
+      sha256 = "0pkpwnk0kfdxsbzsxqlqh8kgif17l5has0mg31g3kyp8lncj89b1";
```

Comment:
Pretty sure this hash is wrong.

```
$ nix-prefetch-url 'https://github.com/pygae/clifford/commit/955d141662c68d3d61aa50a162b39e656684c208.patch'
path is '/nix/store/ya0hpzdm4kd6gaxx0abv7hn1xcadinq0-955d141662c68d3d61aa50a162b39e656684c208.patch'
00m8ias58xycn5n78sy9wywf4wck1v0gb8gzmg40inzdiha93jyz
```

The new hash you've pasted is the hash of the `.diff`, not the hash of the `.patch`.

Reply:
Yeah, because the patch is fetched not the commit...

Reply:
For me this is the new hash because something probably changed or the patch got rebased.